src/Gui/Forms/InitialPageInteractor.cs

```diff
@@ -106,35 +107,44 @@ public override bool LeavePage()
         /// </summary>
         /// <param name="file"></param>
         /// <returns>True if the opened file was a Reko project.</returns>
-        public bool OpenBinary(string file)
+        public bool OpenBinary(string file, string outputDir)
         {
             var ldr = Services.RequireService<ILoader>();
             this.Decompiler = CreateDecompiler(ldr);
             var svc = Services.RequireService<IWorkerDialogService>();
             bool isOldProject = false;
             svc.StartBackgroundWork("Loading program", delegate ()
             {
-                isOldProject = Decompiler.Load(file);
+                isOldProject = Decompiler.Load(file, outputDir);
             });
             if (Decompiler.Project == null)
+            {
                 return false;
+            }
+
+            ProgramResourceGroup prg = Decompiler.Project.Programs[0].Resources;
```

Comment:
Should be `foreach` programs, is not it?

Reply:
Because this method may have opened a project file, you should iterate across all the `Program`s in the `Decompiler.Project`.